single-page.bs

```diff
@@ -741,6 +741,13 @@ urlPrefix: https://www.w3.org/TR/workers/#; spec: WORKERS;
     text: worker processing model; url: processing-model; type: dfn
     text: WorkerGlobalScope; url: workerglobalscope; type: interface
 
+<!--  ****** Longdesc - HTML extension ****** -->
+
+url: https://www.w3.org/TR/html-longdesc/#widl-description-longdesc; type: attribute; for: img;
+    text: longDesc
+url: https://www.w3.org/TR/html-longdesc/; type: element-attr; for: img;
```

Comment:
This creates a bad link, using '#https://' etc. relative to the current spec. as parent, change to urlPrefix corrects it.